example/template.yaml

```diff
@@ -88,6 +88,35 @@ Resources:
             Path: /helloparams
             Method: get
 
+  IdempotencyTable:
+    Type: AWS::Serverless::SimpleTable
+    Properties:
+      TableName: idempotency_table
+      PrimaryKey:
+        Name: id
+        Type: String
+
+  HelloWorldIdempotentFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: HelloWorldFunction
+      Handler: helloworld.AppIdempotency::handleRequest
+      MemorySize: 512
+      Environment: # More info about Env Vars: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#environment-object
+        Variables:
+          POWERTOOLS_LOG_LEVEL: INFO
+          AWS_ENDPOINT_DISCOVERY_ENABLED: false
```

Comment:
I'm not familiar with this. What is it used for?

Reply:
I had some warnings with endpoint discovery in the logs (when you put logs in debug). Found out that with this variable the logs was not there anymore. More info [here](https://stackoverflow.com/questions/59582540/what-is-endpoint-discovery). I'm not sure to fully understand though.